_posts/2020-08-05-cockpit-225.md

```diff
@@ -0,0 +1,75 @@
+slug: cockpit-225
+category: release
+summary: Cockpit and Cockpit Podman news
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 223 and Cockpit Podman version 21.
+
+### Machines: Add support for VM snapshots
+
+Libvirt supports two types of snapshots, [internal and external](https://libvirt.org/formatsnapshot.html),
+but external are not yet in feature parity with the internal. For this reason, cockpit developers decided to go
+with the internal snapshots in the Cockpit UI. Because of using internal snapshots this feature only works with
+qcow2 disk images. VM snapshots are supported for running VMs, in which case the snapshot will combine disk and
+VM memory state, as well as offline VMs, in which case only the disk contents will be saved.
+
+![VM snapshots](/images/machines-snapshots.png){:.screenshot}
+
+### Developer API: Launch and reattach to a long-running process
+
+Cockpit pages should never directly run long-running precious processes such as Software installers or Ansible playbooks,
+as a Cockpit session is not reliably connected for a long time. If you work on such Cockpit page, please take a look
+at the new [long-running process example](https://github.com/cockpit-project/cockpit/tree/master/examples/long-running-process),
+which shows how to do that in a robust way: The process runs in a transient systemd unit, and the page reattaches
+to an already running process, so that the session can be terminated in between.
+
+The logic is implemented in a [JavaScript module](https://github.com/cockpit-project/cockpit/blob/master/pkg/lib/long-running-process.js)
+which you can copy into your own project.
+
+### Podman: Support for pod groups
+
+Alongside images and container now cockpit-podman plugin can also list pods with containers that belong
+into it. Also basic actions like starting or pausing pods are supported.
+
+![Cockpit pod groups](/images/podman_pods.png){:.screenshot}
+
+### Podman: Support checkpoint and restore
+
+Checkpointing a container stops the container and writes the state of all container processes to disk.
+Such a checkpoint can be restored later to resume the container in the exact state as it was at checkpoint time.
+This capability requires [CRIU 3.11](https://www.criu.org/Main_Page) or later installed. Currently
+only the `runc` container runtime is supported; this is only available with `cgroupv1` systems.
+
+![Cockpit podman checkpoint](/images/podman-checkpoint.png){:.screenshot}
+
+_Thanks to Tomáš Glozar for this contribution!_
+
+### Podman: Registry selection in "download image" dialog
+
+Users can pick specific registry to be used when looking for and downloading images.
+
+![Cockpit podman registries](/images/podman-registries.png){:.screenshot}
+
+### Podman: Selected tag removal during deletion
+
+When deleting image that has multiple tags only specific tags can be picked to be removed.
```

Comment:
Deleting images now shows a confirmation dialog where specific tags can be selected for removal.